sdk/core/azure-core/azure/core/_pipeline_client.py

```diff
@@ -65,17 +65,6 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-def _prepare_request(request):
```

Comment:
purposely didn't add anything in the changelog, I'm assuming we don't want to publicize us adding old properties onto our new requests